api/reviews_test.go

```diff
@@ -132,4 +182,72 @@ func TestReviews(t *testing.T) {
 			})
 		})
 	})
+
```

Comment:
Consider writing tests to cover some of the possible error scenarios:
- Book id is missing or empty
- GetBook returns an error
- GetReviews returns an error.

Maybe try writing those tests first, and what you expect to happen before updating the implementation. This is prove your functionality will work in the way you expect it to OR it will work differently and you'll need to make changes 😄